site/src/components/AuthAndFrame/AuthAndFrame.tsx

```diff
@@ -15,14 +17,37 @@ interface AuthAndFrameProps {
  * Wraps page in RequireAuth and renders it between Navbar and Footer
  */
 export const AuthAndFrame: FC<AuthAndFrameProps> = ({ children }) => {
-  const styles = useStyles()
+  const styles = useStyles({})
   const xServices = useContext(XServiceContext)
+  const [authState] = useActor(xServices.authXService)
   const [buildInfoState] = useActor(xServices.buildInfoXService)
+  const [updateCheckState, updateCheckSend] = useActor(
+    xServices.updateCheckXService,
+  )
+
+  useEffect(() => {
+    if (authState.matches("signedIn")) {
+      updateCheckSend("CHECK")
+    } else {
+      updateCheckSend("CLEAR")
+    }
+  }, [authState, updateCheckSend])
 
   return (
     <RequireAuth>
       <div className={styles.site}>
         <Navbar />
+        {updateCheckState.context.show && (
+          <div className={styles.updateCheckBanner}>
+            <Margins>
+              <UpdateCheckBanner
+                updateCheck={updateCheckState.context.updateCheck}
+                error={updateCheckState.context.error}
+                onDismiss={() => updateCheckSend("DISMISS")}
```

Comment:
review(site): Dismissal as any user will prevent notification for showing up until browser window is reloaded, even if logged in/out in-between. This seems like acceptable behavior (managed via XState service).